src/components/FeatureViews/Courses/CourseClasses.js

```diff
@@ -268,234 +273,223 @@ const CourseClasses = () => {
+                    <Typography
+                        variant="body1"
+                        align="left"
+                        className={classes.dataFontDate}
+                    >
+                        {fullName(data.course.instructor.user)}
+                    </Typography>
                 </Grid>
+                <Grid item xs={2} md={4} lg={2} xl={2}>
+                    <Typography
+                        variant="body2"
+                        align="left"
+                        className={classes.alignTitleLeft}
+                    >
+                        Grade
+                    </Typography>
+                    <Typography
+                        variant="body1"
+                        align="left"
+                        className={classes.dataFontDate}
+                    >
+                        {gradeLvl(academicLevel)}
+                    </Typography>
+                </Grid>
+                <Grid item xs={2} md={4} lg={2} xl={2}>
+                    <Typography
+                        variant="body2"
+                        align="left"
+                        className={classes.alignTitleLeft}
+                    >
+                        Subject
+                    </Typography>
+                    <Typography
+                        variant="body1"
+                        align="left"
+                        className={classes.dataFontDate}
+                    >
+                        {courseCategory}
+                    </Typography>
+                </Grid>
+            </Grid>
 
-                <Grid container style={{ marginTop: '2.5em' }}>
-                    <Grid item xs={12} sm={12}>
-                        <ThemeProvider theme={theme}>
-                            <Toolbar disableGutters>
-                                <ChromeTabs
-                                    className={tabSelection()}
-                                    tabs={
-                                        comparison(
-                                            data.parent?.studentList,
-                                            data.enrollments
-                                        )
-                                            ? tabLabels
-                                            : [{ label: 'About Course' }]
+            <Grid container style={{ marginTop: '2.5em' }}>
+                <Grid item xs={12} sm={12}>
+                    <ThemeProvider theme={theme}>
+                        <Toolbar disableGutters>
+                            <ChromeTabs
+                                className={tabSelection()}
+                                tabs={
+                                    comparison(
+                                        data.parent?.studentList,
+                                        data.enrollments
+                                    )
+                                        ? tabLabels
+                                        : [{ label: 'About Course' }]
```

Comment:
oh shoot i forgot this was an older version. work with @clam1987 to figure out what changes should be on here

Reply:
const adminTabs = [
    { label: "About Course", "data-cy": "description" },
    { label: "Announcements", "data-cy": "announcements" },
    { label: "Student Enrolled", "data-cy": "student-enrollment" },
    { label: "Sessions", "data-cy": "sessions" },
    { label: "Attendance", "data-cy": "attendance" }
  ];

  const parentTabWithStudentEnrolledTabs = [
    { label: "About Course" },
    { label: "Announcements" },
    { label: "Student Enrolled" },
    { label: "Sessions" },
  ];

  `const parentNostudentEnrolledTab = [{ label: "About Course" }]`

  const setTabsForAccountTypes = (accountType, studentList, enrollmentArray) => {
    switch(accountType) {
      case "PARENT":
        if(checkIfParentHasStudentEnrolled(studentList, enrollmentArray)) {
          return parentTabWithStudentEnrolledTabs
        }
        return parentNostudentEnrolledTab
      default:
        return adminTabs
    }
  }

<ChromeTabs
       className={
          tabSelection()
          }
        tabs={
          setTabsForAccountTypes(accountType, data.parent?.studentList, data.enrollments)
          }

the code snippet formatted it incorrectly so I just copied and pasted the code into here. Those are the changes I made to 'CourseClasses.js'